ferrocene/doc/user-manual/src/index.rst

```diff
@@ -46,6 +46,14 @@ Ferrocene User Manual
    rustfmt/usage
    rustfmt/config
 
+.. toctree::
+   :numbered:
+   :maxdepth: 2
+   :caption: ferrocene-self-test
```

Comment:
We should rename the pages to `self-test/install` and `self-test/using`, to be consistent with `self-test/error-codes`. I feel like it might also be worth moving that page under this section.